apps/emqx_bridge_kafka/src/emqx_bridge_kafka_impl_consumer.erl

```diff
@@ -183,29 +195,91 @@ on_stop(ResourceId, _State = undefined) ->
         _ ->
             ok
     end;
-on_stop(_ResourceId, State) ->
+on_stop(ConnectorResId, State) ->
     #{
-        subscriber_id := SubscriberId,
+        installed_sources := InstalledSources,
         kafka_client_id := ClientID
     } = State,
-    stop_subscriber(SubscriberId),
+    maps:foreach(
+        fun(_SourceResId, #{subscriber_id := SubscriberId}) ->
+            stop_subscriber(SubscriberId)
+        end,
+        InstalledSources
+    ),
     stop_client(ClientID),
+    ?tp(kafka_consumer_subcriber_and_client_stopped, #{instance_id => ConnectorResId}),
     ok.
 
--spec on_get_status(resource_id(), state()) -> connected | disconnected.
-on_get_status(_ResourceID, State) ->
+-spec on_get_status(resource_id(), connector_state()) -> connected | disconnected.
+on_get_status(_ResourceID, _State) ->
+    ?status_connected.
```

Comment:
I didn't see an API for health checking just the connection without knowing a topic or subscription group...  :see_no_evil:

Reply:
Maybe we can use `brod_sup:find_client(Client)` to check the client is exist .

Reply:
:heavy_check_mark: